Configuration: GNOME, MATE and Screenshots.

NeilWB opened the pull request:
This pull request currently includes new sections and screenshots for  - 

Configuration: Configuring Default Applications
Configuration: Configuring Displays
Configuration: Configuring Mouse and Touchpad
Configuration: Configuring Language
Configuration: Managing Users
Configuration: Enabling Workspaces
Configuration: Configuring Network

------------------------------------

Like a bad rash, I'm back ...

Added information for GNOME & MATE to configuring default applications.  Budgie instructions expanded and tweaked.  Screenshots added.

For #44

NeilWB commented:
OK, I was bored, added a commit for Configuring Displays as well.  

As I've got MATE in a virtual machine, the screenshots show virtual displays.

Let me know if anything needs tweaking. 😄

NeilWB commented:
A commit for the Mouse & Touchpad page.  GNOME, MATE sections and screenshots for the latter.

NeilWB commented:
A commit for the Language page.  I've not added a MATE section as there doesn't seem to be an easy method for changing the language.  As documented here - [https://dev.solus-project.com/T1579](https://dev.solus-project.com/T1579)

No new screenshots required on this one.

NeilWB commented:
A commit for the Managing Users page.  GNOME, MATE etc.  And many screenshots.  

Bob Smith and Jane Doe say hello btw!

JoshStrobl commented:
Thanks for all the improvements. I'll be sure to review these when I get the time, you haven't been forgotten :smile:

NeilWB commented:
MATE Workspace section today following on from @mcritchlow Budgie section.

I've changed the PR topic to something shorter and won't keep changing it now!  Was cluttering up this request.  I've added a list to my first comment showing what I've covered so far.

JoshStrobl commented on configuration/configuring-default-applications/en.md:
```diff
@@ -3,19 +3,58 @@ title = "Configuring Default Applications"
 # Configuring Default Applications
 
-You can configure default applications by going to the Budgie Menu, opening Settings app, and going to the Details section.
+Easily change your default web browser, e-mail client, video player, and more by following the steps detailed below.
```
Doesn't make much sense to have the "by following the steps detailed below", given this would also be shown on the table / list.

JoshStrobl commented on configuration/configuring-default-applications/en.md:
```diff
@@ -3,19 +3,58 @@ title = "Configuring Default Applications"
+
+## GNOME
+
+With GNOME, you will need to open the Applications Overlay using the keyboard shortcut `Super + A` or by clicking the applications icon in the dock and then finding and clicking on "Details".
```
You don't need to specifically use `Super + A`, can do it with just `Super`.

JoshStrobl commented on configuration/configuring-displays/en.md:
```diff
@@ -3,20 +3,64 @@ title = "Configuring Displays"
 # Configuring Displays
 
-You can configure your display(s) by going to the Budgie Menu, opening the Settings app, and clicking the Displays section.
+With Solus it is easy to configure your display and setup how multiple displays will work together.
```
[Should be "set up"](http://www.future-perfect.co.uk/grammar-tip/is-it-setup-set-up-or-set-up/).

JoshStrobl commented on configuration/configuring-displays/en.md:
```diff
@@ -3,20 +3,64 @@ title = "Configuring Displays"
+
+{{< altimg "gnome-display-right-click.jpg" "help-center/configuration/configuring-displays/" >}}
+
+Alternatively, you can open the Applications Overlay using the keyboard shortcut `Super + A` or by clicking the applications icon in the dock and then finding and clicking on "Displays".
```
cough* `Super` cough*

JoshStrobl commented on configuration/configuring-language/en.md:
```diff
@@ -3,20 +3,40 @@ title = "Configuring Language"
 # Configuring Language
 
-You can change your language by going to the Budgie Menu, opening Settings, then clicking Region and Language.
+It is easy to change the language that was selected during the installation of Solus.
```
Doesn't necessarily have to be during the installation of Solus though. Fairly common to do post-install adding of languages for bi-lingual usage.

JoshStrobl commented on configuration/configuring-language/en.md:
```diff
@@ -3,20 +3,40 @@ title = "Configuring Language"
+
+{{< altimg "selecting-language-search.png" "help-center/configuration/configuring-language/" >}}
+
+Once you have selected the language you wish to use, click "Done". You **may** need to logout and login to your computer for it to take affect.
```
Interestingly enough, grammatical use of italics is typically for emphasis, so should be changed to `*may*`

JoshStrobl commented on configuration/configuring-language/en.md:
```diff
@@ -3,20 +3,40 @@ title = "Configuring Language"
+
+## GNOME
+
+With GNOME, open the Applications Overlay using the keyboard shortcut `Super + A` or by clicking the applications icon in the dock and then find and click on "Region and Language".
```
cough* `Super` cough*

JoshStrobl commented on configuration/configuring-language/en.md:
```diff
@@ -3,20 +3,40 @@ title = "Configuring Language"
 {{< altimg "region-and-languages.jpg" "help-center/configuration/configuring-language/" >}}
 
-In this dialog, click Language. The following dialog will appear.
+Start by clicking "Language". This brings up a selection of languages to chose from.
```
Actually requires you to perform the "unlock" action first, so that should be indicated.

NeilWB replied:
Thought it was odd that I missed this step so thought I'd double check.  In the Region & Language settings window, on both Budgie & GNOME, I don't have to unlock it.  This is on clean virtual machine installs.  Maybe I've missed something really obvious? 😕

JoshStrobl replied:
Uhh yea, I'm just gonna say I was hallucinating or something. <sup>Guessing I had the wrong window open, derp.</sup>

JoshStrobl commented on configuration/configuring-language/en.md:
```diff
@@ -3,20 +3,40 @@ title = "Configuring Language"
+
+{{< altimg "region-and-languages.jpg" "help-center/configuration/configuring-language/" >}}
+
+Start by clicking "Language". This brings up a selection of languages to chose from.
```
Actually requires you to perform the "unlock" action first, so that should be indicated.

NeilWB replied:
See earlier reply.

JoshStrobl commented on configuration/configuring-mouse-and-touchpad/en.md:
```diff
@@ -3,30 +3,94 @@ title = "Configuring Mouse and Touchpad"
 # Configuring Mouse and Touchpad
 
-You can configure the settings of your mouse and/or touchpad by going to the Budgie Menu, opening the Settings app, then clicking the Mouse & Touchpad section. You will see either one of the following windows, depending on the device.
+Change your mouse and touchpad settings to suit your taste by using the many options detailed in this guide.
```
"by using the many options detailed in this guide" could be removed.

JoshStrobl commented on configuration/configuring-network/en.md:
```diff
@@ -3,26 +3,90 @@ title = "Configuring Network"
+
+{{< altimg "gnome-top-panel-network-click.jpg" "help-center/configuration/configuring-network/" >}}
+
+Alternatively, you can open the Applications Overlay using the keyboard shortcut `Super + A` or by clicking the applications icon in the dock and then finding and clicking on "Network".
```
cough* `Super` cough*

JoshStrobl commented on configuration/managing-users/en.md:
```diff
@@ -3,25 +3,96 @@ title = "Managing Users"
+To manage your own or other accounts, you first need to authenticate. Click the "Unlock" button in the top right corner and type your password. Note that you are required to be an administrator. If you're the only user on the system, then your account is an administrator by default.
 
-To add a user, click the `+` button in the bottom left corner of the Users window. This will show the Add User window (shown below).
+In the event it is correct, the "Unlock" button will change to "Add User..."
```
"it" should be changed to "the password provided" for further clarification.

JoshStrobl commented on configuration/managing-users/en.md:
```diff
@@ -3,25 +3,96 @@ title = "Managing Users"
+
+{{< altimg "gnome-top-panel-right-click.jpg" "help-center/configuration/configuring-displays/" >}}
+
+Alternatively, you can open the Applications Overlay using the keyboard shortcut `Super + A` or by clicking the applications icon in the dock and then finding and clicking on "Users".
```
cough* `Super` cough*

JoshStrobl commented on configuration/managing-users/en.md:
```diff
@@ -3,25 +3,96 @@ title = "Managing Users"
+
+Fill in the required items and then finish adding the new user by clicking "Add" in the top right corner.
+
+### Removing a user
```
user -> User

JoshStrobl commented on configuration/managing-users/en.md:
```diff
@@ -3,25 +3,96 @@ title = "Managing Users"
+
+Fill in the required items and then finish adding the new user by clicking "Add" in the top right corner.
+
+### Removing a user
```
user -> User

JoshStrobl commented on configuration/managing-users/en.md:
```diff
@@ -3,25 +3,96 @@ title = "Managing Users"
+
+Fill in the required items and then finish adding the new user by clicking "OK" in the bottom right corner.
+
+### Removing a user
```
user -> User

NeilWB commented:
Thanks for checking through all that Josh.  Some daft mistakes and some things I could have phrased better!  I've left a comment above about unlocking the language window.

Have done all the screenshots again with less compression.  I had all the originals so just had to crop a few again.  Have done all of the screenshots I'd committed before this PR as well.

Let me know if I need to make further changes.  😄

JoshStrobl commented:
LGTM. Thanks for the fixes and updating the images (I know that can be time consuming).
